test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -930,102 +928,109 @@ public void VerifyExpectedAndActualBooleanOptionsDictionary(Dictionary<string, D
-            string sqlTable1DefinitionPath = Path.Join(TSqlModelTestFolder, "table1.sql");
-            string sqlTable2DefinitionPath = Path.Join(TSqlModelTestFolder, "table2.sql");
-            const string table1 = @"CREATE TABLE [dbo].[table1]
+[TestFixture]
+public class TSqlModelRequestTests
+{
+    private string TSqlModelTestFolder = Path.Combine("..", "..", "..", "DacFx", "TSqlModels");
+
+    private DacFxService service = new DacFxService();
+
+    [SetUp]
+    public void Create()
+    {
+        Directory.CreateDirectory(TSqlModelTestFolder);
+    }
+
+    [TearDown]
+    public void CleanUp()
+    {
+        Directory.Delete(TSqlModelTestFolder);
+    }
+
+    /// <summary>
+    /// Verify the generate Tsql model operation
+    /// </summary>
+    [Test]
+    public void GenerateTSqlModelFromSqlFiles()
+    {
+        string sqlTable1DefinitionPath = Path.Join(TSqlModelTestFolder, "table1.sql");
+        string sqlTable2DefinitionPath = Path.Join(TSqlModelTestFolder, "table2.sql");
+        const string table1 = @"CREATE TABLE [dbo].[table1]
             (
                 [ID] INT NOT NULL PRIMARY KEY,
             )";
-            const string table2 = @"CREATE TABLE [dbo].[table2]
+        const string table2 = @"CREATE TABLE [dbo].[table2]
             (
                 [ID] INT NOT NULL PRIMARY KEY,
             )";
-            // create sql file
-            File.WriteAllText(sqlTable1DefinitionPath, table1);
-            File.WriteAllText(sqlTable2DefinitionPath, table2);
+        // create sql file
+        File.WriteAllText(sqlTable1DefinitionPath, table1);
+        File.WriteAllText(sqlTable2DefinitionPath, table2);
 
-            var generateTSqlScriptParams = new GenerateTSqlModelParams
-            {
-                ProjectUri = Path.Join(TSqlModelTestFolder, "test.sqlproj"),
-                ModelTargetVersion = "Sql160",
-                FilePaths = new[] { sqlTable1DefinitionPath, sqlTable2DefinitionPath }
-            };
+        var generateTSqlScriptParams = new GenerateTSqlModelParams
+        {
+            ProjectUri = Path.Join(TSqlModelTestFolder, "test.sqlproj"),
+            ModelTargetVersion = "Sql160",
+            FilePaths = new[] { sqlTable1DefinitionPath, sqlTable2DefinitionPath }
+        };
 
-            GenerateTSqlModelOperation op = new GenerateTSqlModelOperation(generateTSqlScriptParams);
-            var model = op.GenerateTSqlModel();
-            var objects = model.GetObjects(DacQueryScopes.UserDefined, ModelSchema.Table).ToList();
+        GenerateTSqlModelOperation op = new GenerateTSqlModelOperation(generateTSqlScriptParams);
+        var model = op.GenerateTSqlModel();
+        var objects = model.GetObjects(DacQueryScopes.UserDefined, ModelSchema.Table).ToList();
 
-            VerifyAndCleanup(sqlTable1DefinitionPath);
-            VerifyAndCleanup(sqlTable2DefinitionPath);
-            Directory.Delete(TSqlModelTestFolder);
+        File.Delete(sqlTable1DefinitionPath);
```

Comment:
And instead of having each test have to cleanup (which again can fail and cause false alarms) I'd just give the test files timestamp names and then they'll all be deleted at the end of the test run anyways.